bundle/pythonic_usage_guidance/pythonic_bundle_access.ipynb

```diff
@@ -0,0 +1,467 @@
+{
```

Comment:
<p>it's a good idea to have this comparison, perhaps the section could be extended to cover more use cases, and describes the previous code vs the new recommended usage</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1501/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Thanks, have added more comparisons and descriptions in the latest commit, could you help review it again?